2-backend-basics/2.0-module-2-overview.md

```diff
@@ -85,7 +85,7 @@ For Windows, Module 2 marks a point that we will change the way files are stored
 Get access to all the files in your WSL Ubuntu instance with this path:
```

Comment:
Could we clarify / add more details here on how this is meant to be a path accessed via the windows file explorer and not a command to be entered in the CLI?